src/containers/Node/NodeStructure/NodeStructure.tsx

```diff
@@ -62,7 +62,7 @@ function NodeStructure({nodeId, className}: NodeStructureProps) {
                 });
             }
         };
-    }, []);
+    }, [scrollContainer]);
```

Comment:
It is useless and misleading to set `ref.current` as `useEffect` dependency.

Reply:
Could you please clarify the issue here? I understand, why `ref` is useless, as it doesn't change. But `ref.current` could be updated and hence could trigger hooks

Reply:
The problem is that ref.current is not updated until component is rendered: here is an article in details: https://medium.com/@teh_builder/ref-objects-inside-useeffect-hooks-eb7c15198780

Reply:
Thanks, I didn't know about this issue. It took me long to process the article, this behaviour isn't obvious at all